src/crawler/crawlers/arso_water_slovenija/crawler.js

```diff
@@ -1,159 +1,173 @@
+            request(url, (err, _res, body) => {
+                if (err) {
+                    console.log(err, "error occured while hitting URL");
+                } else {
+                    const $ = cheerio.load(body);
+                    $('map>area').each(function () {
+                        let link = $(this).attr('href');
+                        link = base_url + link;
+                        links.push(link);
+                    })
+                    resolve(links);
+                }
+            })
+        })
     }
 
-    // Get data and save it in csv file
-    getData(URL) {
-        let data = []
-        let station_name = ''
-        let from_time = 0
-        let self = this
-        request(URL, function (err, res, body) {
-            if(err) {
+    /**
+     * Get data and save it in csv file
+     * @param {String} url 
+     */
+    getData(url) {
+        let data = [];
+        let station_name = '';
+        let from_time = 0;
+        let self = this;
+        request(url, (err, res, body) => {
+            if (err) {
                 console.log(err, "error occured while hitting URL");
-            }
-            else {
-                const $ = cheerio.load(body)
+            } else {
+                const $ = cheerio.load(body);
     
                 $('body>table>tbody>tr>td.vsebina>h1').each(function() { // find name of station
-                    station_name = $(this).text().replace(/ /g, "_").replace(/_-_/g, "-").replace(/\//g, "-")
-                })
-    
-                console.log(station_name)
-                fs.readFile(self.data_file + station_name + '.csv', 'utf-8', function(err, dat) {
+                    station_name = $(this).text().replace(/ /g, "_").replace(/_-_/g, "-").replace(/\//g, "-");
+                });
+
+                fs.readFile(self.data_file + station_name + '.csv', 'utf-8', (err, dat) => {
                     if (err) {
-                        data = self.findData($, data, from_time)
-                        self.saveToFile(data, station_name)
-                    }
-                    else {
-                        from_time = self.checkLastDate(dat)
-                        data = self.findData($, data, from_time)
-                        self.saveToFile(data, station_name)
+                        data = self.findData($, data, from_time);
+                        self.saveToFile(data, station_name);
+                    } else {
+                        from_time = self.checkLastDate(dat);
+                        data = self.findData($, data, from_time);
+                        //console.log(data)
+                        self.saveToFile(data, station_name);
                     }
-                })
+                });
             }    
-        })
+        });
     }
 
-    // save data to file
+    /**
+     * Save data to file.
+     * @param {Array} data 
+     * @param {String} station_name 
+     */
     saveToFile(data, station_name) {
```

Comment:
Use function from CrawlerUtils to save to log file. Add anything to that function if it is needed. Preferred saving format to data lake is ldjson (line-delimited JSON). Try using that one. Check gorivo.si example.